docs/troubleshooting.md

```diff
@@ -0,0 +1,37 @@
+# Troubleshooting
+
+- [Your system can't find Nix](#your-system-cant-find-nix)
+
+## Your system can't find Nix
+
+### Issue
+
+You've run the installer but when you run any Nix command, like `nix --version`, and Nix isn't found:
+
+```shell
+$ nix --version
+bash: nix: command not found
+```
+
+### Likely problem
+
+Nix isn't currently on your `PATH`.
+
+### Potential solutions
+
+1. Start the Nix daemon by running
```

Comment:
That's not what this does. This just sets up various environment variables for Nix to run smoothly (i.e. where it should place its various caches and similar)